code/game/objects/items/weapons/holy_weapons.dm

```diff
@@ -283,7 +283,7 @@
 
 /obj/item/weapon/nullrod/whip
 	name = "holy whip"
-	desc = "What a terrible night to be on Space Station 13."
+	desc = "What a terrible night to be on the Cyberiad!"
```

Comment:
`Space Station 13` is accurate enough here, IMO; the Cyberiad is the station's explicit name, but SS13 is a catchall if you don't intend to use a specific proc, largely because every single spess stayshun out there is SS13.